package.json

```diff
@@ -66,73 +66,80 @@
     "@netlify/functions": "^1.4.0",
-    "@nrwl/cli": "14.5.8",
-    "@nrwl/cypress": "14.5.8",
-    "@nrwl/devkit": "14.5.8",
-    "@nrwl/eslint-plugin-nx": "14.5.8",
-    "@nrwl/js": "14.5.8",
-    "@nrwl/linter": "14.5.8",
-    "@nrwl/storybook": "14.5.8",
-    "@nrwl/web": "14.5.8",
-    "@nrwl/workspace": "15.7.2",
+    "@nrwl/js": "^16.0.0",
+    "@nx/cypress": "^16.0.0",
+    "@nx/devkit": "^16.0.0",
+    "@nx/eslint-plugin": "^16.0.0",
+    "@nx/js": "^16.0.0",
+    "@nx/linter": "^16.0.0",
+    "@nx/storybook": "^16.0.0",
+    "@nx/web": "^16.0.0",
+    "@nx/workspace": "^16.0.0",
     "@rollup/plugin-alias": "^4.0.2",
     "@rollup/plugin-commonjs": "^24.0.0",
     "@rollup/plugin-json": "^6.0.0",
     "@rollup/plugin-node-resolve": "^15.0.1",
     "@rollup/plugin-typescript": "^11.0.0",
-    "@storybook/addon-a11y": "^6.5.10",
-    "@storybook/addon-docs": "^6.5.16",
-    "@storybook/addon-essentials": "6.5.10",
-    "@storybook/addon-interactions": "^6.5.10",
-    "@storybook/builder-webpack5": "^6.5.14",
-    "@storybook/jest": "^0.0.10",
-    "@storybook/manager-webpack5": "^6.5.14",
-    "@storybook/testing-library": "^0.0.13",
-    "@storybook/web-components": "^6.5.10",
+    "@storybook/addon-a11y": "7.4.6",
+    "@storybook/addon-docs": "7.4.6",
+    "@storybook/addon-essentials": "7.4.6",
+    "@storybook/addon-interactions": "7.4.6",
+    "@storybook/addon-mdx-gfm": "^7.4.6",
+    "@storybook/core-server": "^7.4.6",
+    "@storybook/jest": "^0.2.3",
+    "@storybook/testing-library": "^0.2.2",
+    "@storybook/web-components": "^7.4.6",
+    "@storybook/web-components-webpack5": "^7.4.6",
     "@stylelint/postcss-css-in-js": "^0.37.2",
     "@types/globalize": "^1.5.2",
     "@types/jsonapi-serializer": "~3.6.5",
     "@types/node": "18.7.1",
     "@types/request": "^2.48.8",
     "@types/unzipper": "^0.10.5",
     "@types/web": "^0.0.91",
-    "@typescript-eslint/eslint-plugin": "5.53.0",
-    "@typescript-eslint/parser": "5.30.7",
+    "@typescript-eslint/eslint-plugin": "5.62.0",
+    "@typescript-eslint/parser": "5.62.0",
     "@vitest/coverage-c8": "^0.28.5",
     "browserslist": "^4.20.3",
     "chromatic": "^6.5.1",
     "concurrently": "^7.6.0",
     "copyfiles": "^2.4.1",
-    "cypress": "^13.1.0",
+    "cypress": "^13.0.0",
     "cypress-log-to-output": "^1.1.2",
-    "eslint": "8.15.0",
-    "eslint-config-prettier": "8.1.0",
-    "eslint-plugin-cypress": "^2.12.1",
-    "eslint-plugin-prettier": "^3.3.1",
-    "eslint-plugin-storybook": "^0.6.4",
+    "eslint": "^8.46.0",
+    "eslint-config-prettier": "^8.1.0",
+    "eslint-plugin-cypress": "^2.15.1",
+    "eslint-plugin-prettier": "^5.0.1",
+    "eslint-plugin-storybook": "^0.6.15",
     "express": "^4.17.1",
     "husky": "^7.0.2",
     "jest-useragent-mock": "^0.1.1",
     "jsdom": "^20.0.0",
     "lerna": "^6.0.0",
     "mockdate": "^3.0.5",
     "npm-run-all": "^4.1.5",
-    "nx": "14.5.8",
+    "nx": "^16.0.0",
+    "nx-cloud": "16.4.0",
```

Comment:
Do we need nx-cloud?

Reply:
it is recommended as a caching tool that could speedup CI builds, but it shouldn't be in the scope of the current story. Will remove it